jobs/config.json

```diff
@@ -869,6 +869,22 @@
       "sig-instrumentation"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-dind-ipv6": {
```

Comment:
we are inlining these into the prowjob definitions now and deprecating this file
args go after a bare `--` arg to bootstrap in the spec's args, like:
```
args:
- --repo=blah
- --
- --cluster=
- --deployment=kubeadm-dind
...
```
The scenario value goes prior to the `--` like `--scenario=kubernetes_e2e`